case/src/test/kotlin/com/ritense/case/web/rest/CaseTabManagementResourceIntTest.kt

```diff
@@ -34,10 +34,11 @@ import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.http.MediaType
 import org.springframework.security.test.context.support.WithMockUser
 import org.springframework.test.web.servlet.MockMvc
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders
```

Comment:
```suggestion
import org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get
```

Minor thing, but it's nice to be consistent.